sync_integration_suite_test.go

```diff
@@ -51,7 +51,7 @@ var (
 	testConfig        config.Config
 	testSetup         *workflowhelpers.ReproducibleTestSuiteSetup
 
-	session *Session
+	portForwardingSession *Session
```

Comment:
Can we rename this back to what it was "session", I think it can either be `PortforwardingScript` or `bosh`, correct?

Reply:
Bosh is doing only port forwarding.

Reply:
There is another occurrence of session that shadowed this one. I wanted to be more explicit on what it is doing.

Reply:
Clarifying, Bosh is using the `-N` flag.
```
-N' Do not execute a remote command. This is useful for just forwarding ports (protocol version 2 only).
```